sda-commons-server-kafka/src/test/java/org/sdase/commons/server/kafka/health/KafkaWithoutHealthCheckIT.java

```diff
@@ -0,0 +1,120 @@
+                        adminConfig.setAdminClientRequestTimeoutMs(2000);
+                        kafka.setAdminConfig(adminConfig);
+
+                        kafka
+                            .getConsumers()
+                            .put(
+                                CONSUMER,
+                                ConsumerConfig.builder()
+                                    .withGroup("default")
+                                    .withClientId(CONSUMER)
+                                    .addConfig(
+                                        "key.deserializer",
+                                        "org.apache.kafka.common.serialization.LongDeserializer")
+                                    .addConfig(
+                                        "value.deserializer",
+                                        "org.apache.kafka.common.serialization.LongDeserializer")
+                                    .build());
+
+                        kafka
+                            .getProducers()
+                            .put(
+                                PRODUCER,
+                                ProducerConfig.builder()
+                                    .addConfig(
+                                        "key.serializer",
+                                        "org.apache.kafka.common.serialization.LongSerializer")
+                                    .addConfig(
+                                        "value.serializer",
+                                        "org.apache.kafka.common.serialization.LongSerializer")
+                                    .build());
+
+                        kafka
+                            .getListenerConfig()
+                            .put(
+                                LISTENER_CONFIG_ASYNC,
+                                ListenerConfig.builder().withTopicMissingRetryMs(60000).build(1));
+
+                        kafka
+                            .getTopics()
+                            .put(
+                                "topicId1",
+                                TopicConfig.builder()
+                                    .name("topic1")
+                                    .withPartitions(2)
+                                    .withReplicationFactor(2)
+                                    .addConfig("max.message.bytes", "1024")
+                                    .addConfig("retention.ms", "60480000")
+                                    .build());
+                      })
+                  .build());
+
+  @ClassRule
+  public static final TestRule CHAIN = RuleChain.outerRule(KAFKA).around(DROPWIZARD_APP_RULE);
+
+  private KafkaWithoutHealthCheckTestApplication app;
+
+  @Before
+  public void before() {
+    app = DROPWIZARD_APP_RULE.getRule().getApplication();
+  }
+
+  @Test
+  public void healthCheckShouldNotContainKafka() {
+    SortedSet<String> checks = app.healthCheckRegistry().getNames();
+    assertThat(checks).isNotEmpty().doesNotContain("kafkaConnection");
```

Comment:
I'd prefer using a constant defined in `KafkaBundle`.

Reply:
Moved to constant `HEALTHCHECK_NAME` in KafkaBundle.